### ISSUE TITLE
Difficult Grey

### ISSUE THREAD

**Issue by ValentineRain** [ScoreShots/templatemigration#404]
![](https://scoreshotstemplates.s3.amazonaws.com/90.jpg)

# 

* Legacy name: StartingShot 6
* Legacy File: https://scoreshotstemplates.s3.amazonaws.com/90.json
* Legacy Category: DevelopmentTest
* (If Client Requested) Who? 
* (If Client Requested) Still Client? 

* Potentially Problematic Elements: 

* Long-term potential merge candidates 

### Template Types Recommendation:
* Internal
* Post-Game
* Player Highlight
* Game Preview
* Mid-Game
* Social Templates
* Opening Lineup
* Player Quote
* Stats Highlight
* Poll Showcase
* Schedule Showcase
* Bracket Showcase
* Player Signing

### Template Categories Recommendation:
* Baseball
* Basketball
* Field Hockey
* Football
* Golf
* Ice Hockey
* Lacrosse
* Motorsports
* Soccer
* Softball
* Swimming and Diving
* Tennis
* Volleyball
* Wrestling
* X-Country

**Comment by adavila17** [comment]
hockey usage only no need for dynamic steps.

**Comment by rocksprings** [comment]
What is the rationale for retiring this template? I think it looks pretty good.

**Comment by ValentineRain** [comment]
`Development version of an admittedly pretty good template. This copy is no longer needed.`

Please at least read the thread if you're going to do this for every retiring candidate.

**Comment by rocksprings** [comment]
I read the thread but couldn't figure out what it meant. 
Are you indicating it's a duplicate of something we already have?

**Comment by ValentineRain** [comment]
Yes. I am indicating that this _development version_ of the template, stored in a completely different category, was practice for the real one, a proof of concept. It was finished and implemented. This copy is no longer needed.

If you do not understand the way that these items were plainly documented, you probably should not be involved in this process.
